tests/integration/test_api_gateway.py

```diff
@@ -295,7 +362,7 @@ def _test_api_gateway_lambda_proxy_integration(self, fn_name, path):
 
         self.assertTrue(re.match(r'^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}$', source_ip))
 
-        self.assertEqual(request_context['path'], '/' + self.TEST_STAGE_NAME + '/lambda/foo1')
+        self.assertEqual(request_context['path'], '/' + self.TEST_STAGE_NAME + path_with_replace)
```

Comment:
this was previously hardcoded - that was an error. see the change to `get_resource` above.